paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/WalletsState.kt

```diff
@@ -41,7 +42,16 @@ internal data class WalletsState(
             onLinkPressed: () -> Unit,
             isSetupIntent: Boolean
         ): WalletsState? {
-            val link = Link(email = linkEmail).takeIf { isLinkAvailable == true }
+            val link = if (isLinkAvailable == true) {
+                Link(
+                    state = LinkButtonState.create(
+                        linkEmail = linkEmail,
+                        paymentDetails = null
```

Comment:
Not using payment details on ece button for now, just OTP. 

However, the view is ready to display it.